packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.tsx

```diff
@@ -34,6 +35,10 @@ export const PaginationRow: FC<PaginationRowProps> = ({
     }
   };
 
+  const t = useTranslation();
+  const getNumberSelectedLabel = () =>
+    !!numberSelected && `(${numberSelected} ${t('label.selected')})`;
```

Comment:
if you used a parameter in the translation then it might be helpful if a language requires the digit to be after the word. speculative, I know. 

```
"label.number-selected": "({numberSelected} Selected)",
```
and
```  
const getNumberSelectedLabel = () => !!numberSelected && t('label.number-selected', { numberSelected });
```

Reply:
oh, it is checking `!!numberSelected` when being called too, so maybe not required here